io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,411 @@
+    sources:
+      - type: archive
+        url: https://www.prevanders.net/libdwarf-0.11.0.tar.xz
+        sha256: 846071fb220ac1952f9f15ebbac6c7831ef50d0369b772c07a8a8139a42e07d2
+
+  - name: gsl
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
+        sha256: 6a99eeed15632c6354895b1dd542ed5a855c0f15d9ad1326c6fe2b2c9e423190
+
+  - name: glm
+    buildsystem: cmake
+    config-opts:
+     - -DGLM_BUILD_TESTS=OFF
+     - -DBUILD_SHARED_LIBS=ON
+     - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/g-truc/glm/archive/refs/tags/1.0.1.tar.gz
+        sha256: 9f3174561fd26904b23f0db5e560971cbf9b3cbda0b280f04d5c379d03bf234c
+
+  - name: openblas
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - export DYNAMIC_ARCH=1
+      - export USE_OPENMP=1
+      - export NUM_THREADS=56
+      - export TARGET=$(uname -m)
+      - export NO_SVE=1
+      - make CC=gcc FC=gfortran libs netlib shared -j1
+      - make PREFIX=/app install
+      - ln -s /app/lib/libopenblas.so /app/lib/libblas.so
+      - ln -s /app/lib/libopenblas.so /app/lib/liblapack.so
+    sources:
+      - type: archive
+        url: https://github.com/OpenMathLib/OpenBLAS/archive/refs/tags/v0.3.28.tar.gz
+        sha256: f1003466ad074e9b0c8d421a204121100b0751c96fc6fcf3d1456bd12f8a00a1
+
+  - name: bdw-gc
+    buildsystem: autotools
+    config-opts:
+      - --disable-debug
+      - --disable-docs
+      - --disable-dependency-tracking
+    sources:
+      - type: git
+        url: https://github.com/ivmai/bdwgc
+        tag: v8.2.4
+
+  - name: guile
+    buildsystem: autotools
+    config-opts:
+      - --with-libltdl-prefix=/app
+      - --with-libreadline-prefix=/app
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile/guile-2.2.7.tar.xz
+        sha256: cdf776ea5f29430b1258209630555beea6d2be5481f9da4d64986b077ff37504
+
+  - name: g-wrap
+    buildsystem: autotools
+    config-opts:
+      - --disable-Werror
+    sources:
+      - type: archive
+        url: http://savannah.nongnu.org/download/g-wrap/g-wrap-1.9.15.tar.gz
+        sha256: 0ff6e2700e74b457323726f7c2551a466d8ba44339705f6792d7b533145c602a
+
+  - name: guile-cairo
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/guile-cairo/guile-cairo-1.10.0.tar.gz
+        sha256: ae3d7e0f0a8ae93d0a15c692514eb30f2478d407052064976fc59a291cccdd5c
+
+  - name: guile-lib
+    buildsystem: autotools
+    config-opts:
+      - --with-guile-site=yes
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/guile-lib/guile-lib-0.2.7.tar.gz
+        sha256: e4ef3b845f121882c7c0cf04f81a1cb8fd360c6f64b56b868de5546214f904de
+
+  - name: guile-gnome-platform
+    buildsystem: autotools
+    build-options:
+      - cflags: "-Wall"
```

Comment:
is this needed ?

Reply:
Now testing.

Reply:
Without it building the module passed. Deleted.